aks/application/variables.tf

```diff
@@ -102,3 +114,14 @@ variable "github_personal_access_token" {
   default     = null
   description = "Github Personal Access Token (PAT) of github_username"
 }
+
+variable "azure_resource_prefix" {
+  type        = string
+  default     = null
+  description = "Prefix of Azure resources for the service"
+}
+
+variable "azure_enable_monitoring" {
```

Comment:
Can we add a description for this variable? I wonder if it's also worth adding a section to the README for the application module similar to the databases: https://github.com/DFE-Digital/terraform-modules/tree/main/aks/postgres#monitoring